whylog/config/__init__.py

```diff
@@ -103,6 +114,46 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def is_free_parser_name(self, parser_name):
```

Comment:
We have here very similar methods. Maybe remove method above and rename method below? I can give you blacklist to `is_free_parser_name` method as I give it to method `propose_parser_name`. My code will be cleaner then, and you will have one method less.